app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,9 @@
 class PostsController < ApplicationController
-  before_action :authenticate_admin!
+  before_action :load_and_authorize_resource, only: [:destroy]
```

Comment:
- [ ] Please remove only: [:destroy], it doesn't need to be there just leave the :load_and_authorize_resource for

Reply:
resolve issue